support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -17,16 +17,15 @@ Next:
     submit: True
 {% elif eccentricity_control %}
 Next:
-  Run: spectre.Pipelines.Bbh.EccentricityControlMinimal:ecc_control_fit
+  Run: spectre.Pipelines.Bbh.EccentricityControl:ecc_control_fit
   With:
     h5_file: ./BbhReductions.h5
     subfile_name_aha: ApparentHorizons/ControlSystemAhA_Centers.dat
     subfile_name_ahb: ApparentHorizons/ControlSystemAhB_Centers.dat
-    tmin: 3000
-    tmax: 3800
-    angular_velocity_from_xcts: 0.014
-    expansion_from_xcts: -1
+    tmin: 500
+    tmax: None
     output: ecc_control.pdf
+    id_input_file_path: ../../../bbh_id/InitialData.yaml
```

Comment:
Replace `../../../bbh_id/InitialData.yaml` -> `{{ id_input_file_path }}`

Then in `start_inspiral()` where you pass on the `eccentricity_control` boolean to `schedule()` you also pass on `id_input_file_path`.

Reply:
I'm not sure how to do the placeholders since I'm not actually running the whole sim when I'm doing these tests. That's why I used the ../.../../bbh_id/InitialData.yaml line

Reply:
Use the `{{ id_input_file_path }}` placeholder in this file. Then any generated input file will have the full path to the input file inserted, so you can copy some sample data into the directory instead of running the simulation and then run `spectre run-next`. Let's talk in a call if there's anything unclear.